Formula/pueue.rb

```diff
@@ -63,7 +63,11 @@ def plist
     sleep 2
 
     begin
-      mkdir testpath/"Library/Preferences"
+      path = testpath/"Library/Preferences"
+      on_linux do
+        path = testpath/".config"
+      end
+      mkdir path
```

Comment:
Why not just create both directories in `testpath`? It probably doesn't hurt, and limits your change to just adding a line.